archiver/README.md

```diff
@@ -9,3 +9,9 @@ description: |
 ---
 
 Alias for https://webinstall.dev/arc
+
+### Files
+
+```text
```

Comment:
- ALL files sections should include `~/.config/envman/PATH.env` as the first item.
- ALL files sections must have a closing ` ``` `

Reply:
@coolaj86 Sorry for the confusion, despite your comment on PATH.env, while installing the packages I realised it wasn't always modified, that's why I didn't include it. Taking note of your comments and I'll get back to you asap.

Reply:
It's not always modified because it's often just `~/.local/bin/`, which was previously added.

There's only a few special ones that don't require a PATH modification - like the fonts.

Reply:
Makes sense, thanks for clarifying.